tox.ini

```diff
@@ -1,24 +1,17 @@
 [tox]
 envlist =
-	{python27,python34}-{django16,django17,django18}-{lpclient32,lpclient41},
-	python33-{django16,django17,django18}-{lpclient32,lpclient41},
-	python27-django14-lpclient32,
+	{py27,py34,py35}-{django18,django19,django110}-{lpclient41,lpclient5},
 	cov,
 	flake8,
 
 [testenv]
-basepython =
-	python27: python2.7
-	python33: python3.3
-	python34: python3.4
 deps =
 	-r{toxinidir}/requirements-test.txt
-	django14: Django==1.4.22
-	django16: Django==1.6.11
-	django17: Django==1.7.11
 	django18: Django==1.8.16
-	lpclient32: laterpay-client==3.2.1
+	django19: Django==1.9.12
```

Comment:
I dislike pinned Django versions in tox since Django only officially supports the last point releases. How 'bout https://github.com/MarkusH/django-dynamic-forms/blob/master/tox.ini#L21-L24 ?